cli/args.go

```diff
@@ -30,9 +30,9 @@ func InitArgs() (Args, bool) {
 	fs.BoolVar(&args.Verbose, "verbose", false, "Enable verbose logging")
 	fs.BoolVar(&args.Rename, "rename", true, "Keep the original filename or replace it with <basename>.terratag.tf")
 
-	fs.Parse(programArgs)
+	err := fs.Parse(programArgs)
```

Comment:
used https://github.com/kisielk/errcheck to detect files where errors were not handled properly.